en/debug/system_console.md

```diff
@@ -39,17 +50,18 @@ Connect the 6-pos JST SH 1:1 cable to the Dronecode probe or connect the individ
 |5         | SWCLK      |         | N/C   |
 |6         | GND     | 1       | FTDI GND (black)   |
 
-## Pixhawk v1: Wiring the Console
+### Pixhawk v1
```

Comment:
FYI section I plan to remove, with info moved into https://github.com/PX4/px4_user_guide/pull/657

Reply:
Now moved and linked